exercises/exercises.js

```diff
@@ -1,16 +1,65 @@
 // Question 1
-function stripPunctuation() {
-
+function stripPunctuation(phrase) {
+  const alphanums = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'B', 'C', 'D', 'F', 'G', 'H', 'J', 'K', 'L', 'M', 'N', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+  const finalArr = [];
+  for (let i = 0; i < phrase.length; i += 1) {
+    if (alphanums.includes(phrase[i])) {
+      finalArr.push(phrase[i]);
+    }
+  }
+  return finalArr.join('');
 }
 
 // Question 2
-function rotateArray() {
-
+function rotateArray(arrayIn) {
```

Comment:
# Question 2 Score:
## Algorithm
**3**: Algorithm solves the problem.

## Style
**2**: Variables names could be a bit more clear. But mainly, I'm docking you a point for the way you used the `outPut` variable and for using the `typeof` operator instead of the `Array.isArray` method to check if the input argument argument is an array.